source/initial_temperature/S40RTS_perturbation.cc

```diff
@@ -181,6 +181,9 @@ namespace aspect
       // get the degree from the input file (20 or 40)
       unsigned int max_degree = spherical_harmonics_lookup->maxdegree();
 
+      // store the max degree into a constant
+      const unsigned int MAX_degree = max_degree;
```

Comment:
what is the difference between MAX_degree and max_degree? Your variable names and comments should explain that.

Reply:
There should not be two variables with the same name just different upper/lowercase spelling. If they have different semantic meanings, they should have different names.

Reply:
I renamed the variables associated with the SH max degree. Now it should be very clear in the code. Additionally, I also fixed several typos in the input parameters. The degree is in the outer loop of order. Only max degree instead of max order can be specified. The previous version means specifying max degree, but it used 'order' in the input parameter, which can cause confusions to the users. @tjhei @bangerth

Reply:
I also updated the test prm for SAVANI, which truncates the max SH degree for the calculation. @tjhei @bangerth